cmd/earthly/main.go

```diff
@@ -153,8 +153,8 @@ func main() {
 	if envutil.IsTrue("EARTHLY_FULL_TARGET") {
 		padding = conslogging.NoPadding
 	}
-
-	logging := conslogging.Current(colorMode, padding, conslogging.Info)
+	isGitHubActions := envutil.IsTrue("GITHUB_ACTIONS") || cli.Flags().IsGitHubActions
```

Comment:
same as my previous comment, I think the flag should account for the env var being set.
```suggestion
	isGitHubActions := cli.Flags().IsGitHubActions
```